frontend/pages/events.js

```diff
@@ -0,0 +1,361 @@
+                          index={index}
+                        />
+                      );
+                    })}
+                </div>
+              ))}
+
+            {!isMobile && (
+              <PaginationComp
+                totalPages={Math.ceil(selectedPosts.length / postsPerPage)}
+                paginate={paginate}
+                page={currentPage}
+                size="large"
+              />
+            )}
+
+            {/* PAST EVENTS */}
+            <h2>PAST EVENTS</h2>
+            <div className={styles.eventCategories}>
+              <div className={styles.contentCategories}>
+                {Object.keys(categories)
+                  .sort()
+                  .map((category) => {
+                    const chipColour =
+                      selectedCategory === categories[category]
+                        ? "#e85f5c"
+                        : "#7F7F7F";
+                    return (
+                      <Chip
+                        key={category}
+                        size="medium"
+                        label={category}
+                        className={classes.chip}
+                        style={{
+                          backgroundColor: chipColour,
+                        }}
+                        onClick={() => {
+                          setSelectedCategory(categories[category]);
+                          filterContent(categories[category]);
+                        }}
+                      />
+                    );
+                  })}
+              </div>
+
+              <Timeline
+                margin={"3%"}
+                page={"events"}
+                step={50}
+                valueToYear={valueToYear}
+                marks={marks}
+                updateYear={handleYear}
+              />
+
+              <div>
+                {emptyCategory === true && (
+                  <p id={styles.emptyMessage}>
+                    Keep a lookout for more events!
+                  </p>
+                )}
+              </div>
+            </div>
+
+            <div className={styles.eventsLoadingContainer}>
+              {loadingPast && (
+                <CircularProgress
+                  variant="indeterminate"
+                  size={50}
+                  thickness={5}
+                  id={styles.eventsLoading}
+                />
+              )}
+            </div>
+
+            <div className={styles.pastEventsContainer}>
+              {!loadingPast &&
+                Object.keys(pastSelectedPosts)
+                  .reverse()
+                  .map((key) => {
+                    const numEvents = React.Children.count(
+                      getTermEvents(pastSelectedPosts[key])
+                    );
+                    if (numEvents > 0) {
+                      return (
+                        <div key={key}>
+                          <h3 className={styles.termColour}>
+                            TERM {key.replace("term", "")}
```

Comment:
Term text doesn't show up for light mode :( 
![image](https://user-images.githubusercontent.com/80564580/176900504-5b34ec7f-b42c-460b-8621-4092927dd13b.png)

Reply:
Done !!